tests/pytests/unit/cloud/clouds/test_dimensiondata.py

```diff
@@ -0,0 +1,190 @@
+"""
+    :codeauthor: `Anthony Shaw <anthonyshaw@apache.org>`
+
+    tests.unit.cloud.clouds.dimensiondata_test
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+
+import pytest
+
+from salt.cloud.clouds import dimensiondata
+from salt.exceptions import SaltCloudSystemExit
+from salt.utils.versions import LooseVersion
+from tests.support.mock import MagicMock
+from tests.support.mock import __version__ as mock_version
+from tests.support.mock import patch
+
+try:
+    import libcloud.security
+
+    HAS_LIBCLOUD = True
+except ImportError:
+    HAS_LIBCLOUD = False
+
+
+VM_NAME = "winterfell"
```

Comment:
Was going to suggest that this should be a fixture, but I don't see where it's being used.